substrate/runtime/staking/src/lib.rs

```diff
@@ -451,6 +430,42 @@ impl<T: Trait> Module<T> {
 		}
 	}
 
+	/// Set the free balance on an account to some new value.
+	///
+	/// Same as [`set_free_balance`], but will create a new account.
+	///
+	/// [`set_free_balance`]: #method.set_free_balance
+	pub fn set_free_balance_creating(who: &T::AccountId, balance: T::Balance) -> bool {
```

Comment:
document what the return bool means?

Reply:
yeah, good idea. We should also add docs for `set_free_balance`

Reply:
or another option is what I described in https://github.com/paritytech/polkadot/pull/345#discussion_r203054535